backend/src/util/verify.ts

```diff
@@ -10,7 +10,6 @@ initializeApp({
 
 const log = Logger.getLogger('verification')
 
-// function used to determine user authentication
 export default async (token: string) => {
```

Comment:
Could add this comment back, especially because its an unnamed function it would be nice to have something that indicates what it does without having to rely on the file name